centinel/experiments/baseline_linear.py

```diff
@@ -88,6 +88,7 @@ def run_file(self, input_file):
         pcap_results = {}
         pcap_indexes = {}
         url_index = 0
+        index_row = None
         comments = ""
 
         # we may want to make this threaded and concurrent
```

Comment:
this entire script would be *way* more readable if you use the csv.DictReader. It's the same as what you are doing, but you get a dictionary instead of an array, so people can see why you are accessing the each element in the array. This code is basically impossible to read, so if you don't use csv.DictReader, you should add a comment explaining what each row is.